controllers/drplacementcontrol_controller.go

```diff
@@ -438,6 +520,8 @@ type DRPCInstance struct {
 	userPlacementRule    *plrv1.PlacementRule
 	drpcPlacementRule    *plrv1.PlacementRule
 	mwu                  rmnutil.MWUtil
+	schedule             *string
+	replclassselect      *metav1.LabelSelector
```

Comment:
why isn't this variable following the camel case convention?

Reply:
(Future) Just a note here, the DRPCInstance is becoming huge.  We should refactor it.